app/upgrades.go

```diff
@@ -127,6 +129,14 @@ var handlers = map[string]appUpgrade{
 	},
 	"green": {
 		Handler: func(app *App, ctx sdk.Context, plan upgradetypes.Plan) (module.VersionMap, error) {
+			app.IBCKeeper.ClientKeeper.IterateClients(ctx, func(clientId string, state exported.ClientState) bool {
+				tc, ok := (state).(*ibcctmtypes.ClientState)
+				if ok {
```

Comment:
could potentially check if already true and skip the set, could help if we have a lot of them :notsureif:..but probably not required

Reply:
I think there are only a half dozen total connections we are iterating over here... seems fine to brute force here.

Reply:
True, however we have very few ibc connections and the time required to do a state update should be very minimal so I don't know that it would be necessary... 🤷‍♂️